include/commissioner/network_data.hpp

```diff
@@ -52,6 +55,49 @@ static constexpr uint8_t kMlrStatusNoResources = 4;
 static constexpr uint8_t kMlrStatusNotPrimary  = 5;
 static constexpr uint8_t kMlrStatusFailure     = 6;
 
+/**
+ * Extended PAN Id wrapper
+ */
+struct xpan_id
```

Comment:
Why do we need to define a struct for simply an integer?

Reply:
Here and for the PAN below the struct was used as a class with public default access. This allows us strict type control with transparent type casting in places where we need it. It is impossible to have that level of the type checking with generic integer.
On the other hand it doesn't introduce additional memory or other runtime requirements (compared to integer) as there is no virtual function.